ctrl.c

```diff
@@ -1318,9 +1319,9 @@ void pv_ctrl_free_cmd(struct pv_cmd *cmd)
 static int pv_ctrl_init(struct pv_init *this)
 {
 	struct pantavisor *pv = pv_get_instance();
-	char path[PATH_MAX];
+	char path[107];
```

Comment:
why exactly 107?

Reply:
is the sun_path size. Added that as a comment